kea_exporter/kea.py

```diff
@@ -105,6 +105,12 @@ def setup_dhcp4_metrics(self):
                 'Packets received',
                 ['operation']),
 
+            # Allocation
+            'allocation_fail': Gauge(
+                f'{self.prefix_dhcp4}_allocation_fail_total',
```

Comment:
```suggestion
                f'{self.prefix_dhcp4}_allocations_failed_total',
```

I think that's more in line with other metrics. What do you think?